content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -40,12 +40,12 @@ For information on how to create a cluster with kubeadm once you have performed
 -->
 * 一台兼容的 Linux 主机。Kubernetes 项目为基于 Debian 和 Red Hat 的 Linux
   发行版以及一些不提供包管理器的发行版提供通用的指令
-* 每台机器 2 GB 或更多的 RAM （如果少于这个数字将会影响你应用的运行内存）
+* 每台机器 2 GB 或更多的 RAM（如果少于这个数字将会影响你应用的运行内存）
 * 2 CPU 核或更多
-* 集群中的所有机器的网络彼此均能相互连接(公网和内网都可以)
+* 集群中的所有机器的网络彼此均能相互连接（公网和内网都可以）
 * 节点之中不可以有重复的主机名、MAC 地址或 product_uuid。请参见[这里](#verify-mac-address)了解更多详细信息。
-* 开启机器上的某些端口。请参见[这里](#check-required-ports) 了解更多详细信息。
-* 禁用交换分区。为了保证 kubelet 正常工作，你 **必须** 禁用交换分区。
+* 开启机器上的某些端口。请参见[这里](#check-required-ports)了解更多详细信息。
+* 禁用交换分区。为了保证 kubelet 正常工作，你**必须**禁用交换分区。
```

Comment:
The spaces before and after a bold symbol are mostly for easy rendering though the preview is good this time.
See preview: https://deploy-preview-35534--kubernetes-io-main-staging.netlify.app/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm/